test/customer_repository_test.rb

```diff
@@ -1,6 +1,5 @@
-require_relative 'test_helper'
+require './test/test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.